x-pack/test/cases_api_integration/security_and_spaces/tests/trial/cases/push_case.ts

```diff
@@ -48,504 +49,560 @@ import {
   secOnlyRead,
   superUser,
 } from '../../../../common/lib/authentication/users';
+import { RecordingServiceNowSimulator } from '../../../../../alerting_api_integration/common/fixtures/plugins/actions_simulators/server/servicenow_simulation';
+import { loginUsers } from '../../../../common/lib/user_profiles';
 
 // eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext): void => {
   const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
   const es = getService('es');
   const esArchiver = getService('esArchiver');
+  const security = getService('security');
 
   describe('push_case', () => {
-    const actionsRemover = new ActionsRemover(supertest);
-    let serviceNowSimulatorURL: string = '';
-    let serviceNowServer: http.Server;
-
-    before(async () => {
-      const { server, url } = await getServiceNowSimulationServer();
-      serviceNowServer = server;
-      serviceNowSimulatorURL = url;
-    });
-
-    afterEach(async () => {
-      await deleteAllCaseItems(es);
-      await actionsRemover.removeAll();
-    });
-
-    after(async () => {
-      serviceNowServer.close();
-    });
+    describe('incident recorder server', () => {
```

Comment:
This isn't as bad as it looks. I wrapped the old tests in a describe block and created this one that uses the recording servicenow simulator so that I can check the created incident.